modules/gdscript/tests/scripts/completion/foo.notest.gd

```diff
@@ -0,0 +1,6 @@
+extends Node
```

Comment:
I don't like this file called `notest` when it is an actual test. If I understand it correctly this is so the regular test runner ignore this file, but in this case it's better to hard code the `completion` folder to be excluded. There might be completion tests where the `notest` is required (for instance, when making an auxiliary script to see if its properties are listed) so it would be better if the `notest` tag is respected for completion tests as well.

Reply:
Yeah I also thought about this but in the end decided against it since the same approach was choose for the LSP tests. I wonder whether it would make sense to add some naming patter for folders like this to prevent hard coding.

Maybe `completion.norun` and `lsp.norun`. But hard coding could be fine in the context of tests.

Reply:
Hard coded the test runner to ignore `completion` and also did the same for `lsp` to keep it consistent.